src/ServiceControl.Monitoring/ServiceControl.Monitoring.csproj

```diff
@@ -12,11 +12,26 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ServiceControl.Monitoring</RootNamespace>
     <AssemblyName>ServiceControl.Monitoring</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
depends on whether you need to use some of the 4.6.x changes or not. Otherwise it would be more appropriate to use 4.5.2 as the core so you won't force users to upgrade just because they want to use this plugin?